sdk/examples/xo_python/sawtooth_xo/xo_client.py

```diff
@@ -19,6 +19,7 @@
 import requests
 import yaml
 
+
```

Comment:
Does this pass lint? I think there should only be one line break between imports.